package.json

```diff
@@ -33,31 +33,40 @@
     "prelint": "cd samples; npm link ../; npm i"
   },
   "dependencies": {
+    "@octokit/rest": "^18.0.0",
+    "@types/node": "^14.0.13",
+    "@types/pino": "^6.3.0",
+    "@types/supports-color": "^5.3.0",
     "google-gax": "^1.14.1",
-    "protobufjs": "^6.8.0"
+    "pino": "^6.3.2",
+    "protobufjs": "^6.8.0",
+    "supports-color": "^7.1.0"
```

Comment:
similarly don't see us using `supports-color` (I would also bet this might be something handled for you by `pino` when logging).

In general, it's best to avoid any dependencies we're not using directly.